config/config.yaml.tftpl

```diff
@@ -961,6 +1209,14 @@ bigquery:
       dataset: "auto_audience_segmentation"
       stored_procedure: "auto_audience_segmentation_inference_preparation"
   procedure:
+    aggregated_value_based_bidding_training_preparation:
+      project_id: "${project_id}"
+      dataset: "aggregated_vbb"
+      name: "aggregated_value_based_bidding_training_preparation"
+      mds_project_id: "${mds_project_id}"
+      mds_dataset: "${mds_dataset}"
+      view_name: "aggregated_value_based_bidding_training_full_dataset"
+      expiration_duration_hours: 168
```

Comment:
in the procedure itself it saids expire after 48 hours, but here its 168

Reply:
Fixed to 7d.